### DISCUSSION TITLE
FNIRSI GC-01: HV voltage settings for J321 tube

### DISCUSSION THREAD

**Discussion by jackhen** [Gissio/radpro#42] [Q&A] [answered]
I got my tube from Taobao. I ordered a J321 2022. I did the Rad FW, removed the resister. (My GC-01 came with a tube with wires coming from one end-no markings). Everything went well and devices booted fine.  So in setup I made sure J321 selected and set HV profile as  freq=1.25kHz and duty=25%. I don't really know what setting to use here. Is this right? Dead time is =off.  

My question is  what are the right HV profile to set? BTW right now my meter reads 0.140 uSv and that seems not correct. Too high just for a background reading.

Please tell me the setup settings for this tube.

**Comment by Gissio** [comment, marked as answer]
The original firmware should set up 440 V, which works perfectly with the J321. So you should just use the factory default profile.

***NEVER EVER*** set up an HV profile without measuring the high voltage at the tube. I am not responsible if you break your device from overcurrent (the power transistor) or overvoltage (the tube).
